pkg/kapp/resources/resources.go

```diff
@@ -520,13 +539,15 @@ func (c *ResourcesImpl) assumedAllowedNamespaces() ([]string, error) {
 	nsList, err := c.coreClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
 	if err != nil {
 		if errors.IsForbidden(err) {
-			if len(c.fallbackAllowedNamespaces) > 0 {
-				return c.fallbackAllowedNamespaces, nil
+			if len(c.opts.FallbackAllowedNamespaces) > 0 {
+				return c.opts.FallbackAllowedNamespaces, nil
 			}
 		}
-		return nil, fmt.Errorf("fetching all namespaces: %s", err)
+		return nil, fmt.Errorf("Fetching all namespaces: %s", err)
 	}
 
+	c.logger.Info("Falled back to checking each namespace separately (much slower)")
```

Comment:
nit, I think it should be 
```suggestion
	c.logger.Info("Fell back to checking each namespace separately (much slower)")
```
(minor grammar nit)

Reply:
used Falling back to.....